### ISSUE TITLE
Unable to remove/find some of the nodes

### ISSUE THREAD

**Issue by Restuta** [alexbol99/flatten-interval-tree#15]
Test setup:

```js
const IntervalTree = require('@flatten-js/interval-tree').default
const intervalTree3 = new IntervalTree()

intervalTree3.insert([2, 5], 10)
intervalTree3.insert([2, 5], 20)
intervalTree3.insert([2, 5], 30)

if (!intervalTree3.exist([2, 5], 10)) {
  console.log('item does not exist')
}
```

Runkit https://runkit.com/embed/mv6xfvsl298g

Specifically it can't find what used to be a "root" node. I debugged code a little bit and it doesn't seem like node with value 10 is part of the traversal. It seems that the issue is somewhere around this line:

https://github.com/alexbol99/flatten-interval-tree/blob/master/src/classes/intervalTree.js#L457

But I am not certain.

**Comment by alexbol99** [comment]
Hello, Anton,
Bug fixed in v1.0.11.
Thank you for collaboration.

**Comment by Restuta** [comment]
Wow, quick turnaround. Thank you so much for your contributions! 🎉
